### PR TITLE
Traitor Silicon Blacklist Switcheroo 2: Dynamic Goof

### PR DESCRIPTION
**Pull request by jknpj**
Basically removes start/mid/late traitor AI and swaps them with cyborgs so they can be traitor again.

🆑 
* tweak: AIs can no longer be traitor. Cyborgs can be traitor again.

### CODE REVIEW

**Comment by ShiftyRail** [comment]
Wait, actually hold on
Latejoin sec offs can't be inflitrators ?

This is a tragedy

**Comment by SonixApache** [comment]
Wait, does this mean sec and silicons are low-chance traitors, or zero-chance traitors?

**Comment by jknpj** [comment]
Pretty much we now live in a world where AIs can be traitor but not security/captain

**Comment by SonixApache** [comment]
how horrifying

**Comment by ShiftyRail** [comment]
>Pretty much we now live in a ~~world~~ society where AIs can be traitor but not security/captain

**Comment by Spessgontran** [comment]
>Removing Traitor AI to swap it with cyborg traitor
>Not simply allowing cyborgs AND mommis to be traitor just like they used to
One job. Cyborgs and Mommis used to have the chance of becoming Traitor, but was removed because of people bitching. Even if the AI was a traitor himself, borgs still had a chance to become one themselves later on

Beside, traitor AI is funner than being malf because you can just have fun doing stupid shit instead of playing the long boring game of APC handling

**Comment by DamianX** [comment]
fix it

**Comment by jknpj** [comment]
fug

File: code/datums/gamemode/dynamic/dynamic_rulesets_latejoin.dm [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	name = "Syndicate Infiltrator"
 	role_category = ROLE_TRAITOR
 	protected_from_jobs = list("Security Officer", "Warden", "Head of Personnel", "Detective", "Head of Security", "Captain", "Merchant")
-	restricted_from_jobs = list("AI","Cyborg","Mobile MMI")
+	restricted_from_jobs = list("AI", "Cyborg","Mobile MMI")
```
Comment (jknpj) [inline code/datums/gamemode/dynamic/dynamic_rulesets_latejoin.dm]: epic

Comment (SonixApache) [reply code/datums/gamemode/dynamic/dynamic_rulesets_latejoin.dm]: 🤔

**Pull request merged**